src/components/common/ColonyActionsTable/partials/ActionDescription/ActionDescription.tsx

```diff
@@ -42,8 +41,9 @@ const ActionDescription: FC<ActionDescriptionProps> = ({
     true,
   );
 
-  const { expenditure, loadingExpenditure } =
-    useGetExpenditureData(expenditureId);
+  const { expenditure, loadingExpenditure } = useGetExpenditureData(
+    action.streamingPaymentId,
```

Comment:
This change breaks the description of all expenditure related actions (advanced, split, staged payment) in the activity feed.